Mathlib/Data/ENat/Lattice.lean

```diff
@@ -45,7 +45,19 @@ lemma coe_iSup : BddAbove (range f) → ↑(⨆ i, f i) = ⨆ i, (f i : ℕ∞)
 @[norm_cast] lemma coe_iInf [Nonempty ι] : ↑(⨅ i, f i) = ⨅ i, (f i : ℕ∞) :=
   WithTop.coe_iInf (OrderBot.bddBelow _)
 
-variable {s : Set ℕ∞}
+lemma iInf_toNat : (⨅ i, (f i : ℕ∞)).toNat = ⨅ i, f i := by
+  cases isEmpty_or_nonempty ι
+  · simp [iInf_coe_eq_top.mpr ‹_›]
```

Comment:
I think a simp lemma saying `⨅ i, (f i : ℕ∞) = ⊤` in the case that iota is empty would be sensible, and also simplify this bit of the proof